gen/definitions/internal_user.yaml

```diff
@@ -62,27 +63,38 @@ attributes:
     type: String
     description: Last name of the internal user
     example: Doe
+  - model_name: identityGroups
+    data_path: [InternalUser]
+    type: String
+    description: CSV of identity group IDs.
```

Comment:
What is "CSV"? If it means "comma separated" then maybe change this to:
```
description: Comma separated list of identity group IDs.
```

Reply:
This is how it was documented in ERS SDK documentation, yes it means comma separated values. I changed it as per your proposal